chapter_4/service_catalogs_and_user_self-service_portal.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction to ManageIQ services
 
- Services are one of the most interesting feature within ManageIQ. Services can include one or more virtual machines, instances etc. that can be deployed across hybrid environments.
+ Services are one of the most interesting feature within ManageIQ. Services can include one or more virtual machines, instances, etc. that can be deployed across hybrid environments.
 
 ### User self-service portal
 
```

Comment:
Please make the link click able.

Reply:
@KhushbuB , If I am not wrong then then you mean this link -> 
" https:// your_manageiq_appliance / self_service/ " .
 This link is actually not supposed to be clickable.
 The i.p address of user's ManageIQ appliance has to be replaced with <your_manageiq_appliance>.

Reply:
Oops i got it. Sorry :(

Reply:
It's alright :)